pykafka/protocol.py

```diff
@@ -137,8 +137,9 @@ class Message(Message, Serializable):
           Value => bytes
 
     :class:`pykafka.protocol.Message` also contains `partition` and
-    `partition_id` fields. Both of these have meaningless default values when
-    :class:`pykafka.protocol.Message` is used by the producer.
+    `partition_id` fields. Both of these have meaningless default values. When
+    :class:`pykafka.protocol.Message` is used by the producer, `partition_id`
+    identifies the Message's destination partition.
```

Comment:
Thanks, yes, that should make a really nice improvement. I'd never had a good use for `__slots__` before and had all but forgotten about them. Shall I just do that within this pull request? (I guess this is touching so many bits already, I might as well?)